docs/getting_started/logging.rst

```diff
@@ -25,5 +28,12 @@ similar to the following:
    Data: None
    Params: {'raw_json': 1}
    Response: 200 (876 bytes)
+   
+Furthermore, any API ratelimits will produce this output on top of the existing
```

Comment:
Should we clarify which ratelimit this is?

Reply:
Probably. There seems to be a different set of ratelimits on POST actions than normal actions.